navigation/ca_hector/launch/geotiff_mapper.launch

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+
+<!-- Launchfile to save the map generated with Hector SLAM in Tiff format
+as well as the robot's trajectory. This is extracted and adapted from
+https://github.com/tu-darmstadt-ros-pkg/hector_slam/blob/melodic-devel/hector_geotiff/launch/geotiff_mapper.launch
+-->
+
+<launch>
+
+  <node pkg="hector_trajectory_server" type="hector_trajectory_server" name="hector_trajectory_server" output="screen">
+    <rosparam file="$(find ca_hector)/config/hector_trajectory_server.yaml" command="load"/>
+  </node>
+
+  <node pkg="hector_geotiff" type="geotiff_node" name="hector_geotiff_node" output="screen" launch-prefix="nice -n 15">
+    <rosparam file="$(find ca_hector)/config/hector_geotiff.yaml" command="load"/>
+    <!-- Path where the geotiff file will be stored -->
+    <param name="map_file_path" value="$(find ca_hector)/maps"/>
+
```

Comment:
I don't know where is this comment, but `map_file_base_name` should take the value of `env` that you can find in `navigate.launch`.

```suggestion
<param name="map_file_base_name" value="$(arg env)"/>
```

Reply:
done!